Loop/View Models/SettingsViewModel.swift

```diff
@@ -81,7 +81,8 @@ public class SettingsViewModel: ObservableObject {
     let criticalEventLogExportViewModel: CriticalEventLogExportViewModel
     let therapySettings: () -> TherapySettings
     let pumpSupportedIncrements: (() -> PumpSupportedIncrements?)?
-    let syncPumpSchedule: (() -> PumpManager.SyncSchedule?)?
+    let syncPumpSchedule: (() -> SyncSchedule?)?
```

Comment:
This pattern of taking a closure that returns a closure is redundant; we should simplify here.

Reply:
I merged in LoopKit before I saw this comment. I'll add a TODO and tackle this next time.